src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import java.util.Comparator;
+import java.util.List;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+
+/**
+ * A component of the chatBot Duke that returns his response as a string.
+ */
+public class Ui {
```

Comment:
Have you considered making all the methods in this class static, considering `Ui` doesn't have any non-static attributes?